README.md

```diff
@@ -0,0 +1 @@
+## A system/status icon tray is required for this package to function correctly. 
```

Comment:
No one is ever going to read this. It should be mentioned in appdata.

Reply:
done